docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.6'
+services:
+  postgres:
+    image: postgres:alpine
+    environment:
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - '5433:5432'
```

Comment:
Is `5433` intended or a typo?

Reply:
good catch. this PR is missing this file https://github.com/nearform/the-fastify-workshop/blob/master/postgrator.json

Reply:
Good catch @Eomm!

@simoneb postgrator.json is already in the repo